sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -1447,6 +1447,15 @@ directive:
     $.BlobExpiryOptions["x-az-public"] = false;
 ```
 
+### Make BlobPropertiesInternal internal
```

Comment:
Oh nice, you already made BlobPropertiesInternal internal.  Run `sdk/storage/generate.ps1` so this takes effect in the generated code, and  `eng/scripts/Export-API.ps1` so the change takes effect in `api/Azure.Storage.Blobs.netstandard2.0.cs`